neo/Cryptography/Base58.cs

```diff
@@ -7,16 +7,8 @@ namespace Neo.Cryptography
 {
     public static class Base58
     {
-        /// <summary>
-        /// base58编码的字母表
```

Comment:
I prefer translate the comments to english than only remove the chinese version

Reply:
Will do it later.